db-service/lib/cql-functions.js

```diff
@@ -99,6 +99,13 @@ const StandardFunctions = {
    * @returns {string}
    */
   matchesPattern: (x, y) => `(${x} regexp ${y})`,
+  /**
+   * Generates SQL statement that matches the given string against a regular expression
+   * @param {string} x
+   * @param {string} y
+   * @returns {string}
+   */
+  matchespattern: (x, y) => `(${x} regexp ${y})`,
```

Comment:
This function is not defined in the other databases ?

Reply:
Isn't this the generic cql functions that we introduce?

Reply:
Yes, but these are the database specific translations into SQL. I checked and it seems that this currently only works for sqlite.

So this PR should add a test using the function and the posters and HANA versions.

Reply:
Got it, will do.